manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "structured-data",
-  "version": "0.5.0",
+  "version": "0.6.0-beta.0",
```

Comment:
This shouldn't be committed, right?

Reply:
It was committed because I used the `vtex release` command. The publish bot will update it when the PR is merged